pkg/log/default.go

```diff
@@ -0,0 +1,236 @@
+package log
+
+import (
+	"sync"
+	"io"
+	"log"
+	"os"
+	"fmt"
+)
+
+var (
+	logLock       sync.Mutex
+	defaultLogger *Logger
+)
+
+type Level int
+
+const (
+	initText         = "Logger Init wasn't called"
+	flags            = log.Ldate | log.Lmicroseconds | log.Lshortfile
+	FatalLevel Level = iota
+	ErrorLevel
+	WarnLevel
+	InfoLevel
+	DebugLevel
+)
+
+type Logger struct {
```

Comment:
may be it'd be better to have an interface for logger, so we can replace it with another impl in tests.